text/stacks/rfcs/0000-template.md

```diff
@@ -1,23 +0,0 @@
-# {{TITLE: a human-readable title for this RFC!}}
```

Comment:
@sophiewigmore @brayanhenao This template didn't seem very different from the top-level template so I assumed it was okay to remove. Happy to add it back if you want stacks RFCs to follow a different stacks-team specific template.